java/statestore/src/test/java/sleeper/statestore/transactionlog/TransactionLogSnapshotTestBase.java

```diff
@@ -0,0 +1,224 @@
+                .partitionsLogStore(partitionTransactionStoreByTableId.get(table.get(TABLE_ID)))
+                .build();
+        stateStore.fixFileUpdateTime(DEFAULT_UPDATE_TIME);
+        stateStore.fixPartitionUpdateTime(DEFAULT_UPDATE_TIME);
+        return stateStore;
+    }
+
+    protected DynamoDBTransactionLogSnapshotMetadataStore snapshotStore(TableProperties table) {
+        return new DynamoDBTransactionLogSnapshotMetadataStore(instanceProperties, table, dynamoDBClient);
+    }
+
+    protected TableProperties createTable(String tableId, String tableName) {
+        TableProperties tableProperties = createTestTableProperties(instanceProperties, schema);
+        tableProperties.set(TABLE_ID, tableId);
+        tableProperties.set(TABLE_NAME, tableName);
+        tableProperties.set(STATESTORE_CLASSNAME, DynamoDBTransactionLogStateStore.class.getName());
+        fileTransactionStoreByTableId.put(tableId, new InMemoryTransactionLogStore());
+        partitionTransactionStoreByTableId.put(tableId, new InMemoryTransactionLogStore());
+        return tableProperties;
+    }
+
+    protected TransactionLogSnapshotMetadata filesSnapshot(TableProperties table, long transactionNumber) {
+        return TransactionLogSnapshotMetadata.forFiles(getBasePath(instanceProperties, table), transactionNumber);
+    }
+
+    protected TransactionLogSnapshotMetadata partitionsSnapshot(TableProperties table, long transactionNumber) {
+        return TransactionLogSnapshotMetadata.forPartitions(getBasePath(instanceProperties, table), transactionNumber);
+    }
+
+    protected boolean filesSnapshotFileExists(TableProperties table, long transactionNumber) throws IOException {
+        return fs.exists(new Path(filesSnapshot(table, transactionNumber).getPath()));
+    }
+
+    protected boolean partitionsSnapshotFileExists(TableProperties table, long transactionNumber) throws IOException {
+        return fs.exists(new Path(partitionsSnapshot(table, transactionNumber).getPath()));
+    }
+
+    protected void deleteFilesSnapshotFile(TableProperties table, long transactionNumber) throws Exception {
+        fs.delete(new Path(filesSnapshot(table, transactionNumber).getPath()), false);
+    }
+
+    protected void deletePartitionsSnapshotFile(TableProperties table, long transactionNumber) throws Exception {
+        fs.delete(new org.apache.hadoop.fs.Path(partitionsSnapshot(table, transactionNumber).getPath()), false);
+    }
+
+    protected Stream<String> tableFiles(TableProperties tableProperties) throws Exception {
+        Path tableFilesPath = new Path(getBasePath(instanceProperties, tableProperties));
+        if (!fs.exists(tableFilesPath)) {
+            return Stream.empty();
+        }
+        RemoteIterator<LocatedFileStatus> iterator = fs.listFiles(tableFilesPath, true);
+        List<String> files = new ArrayList<>();
+        while (iterator.hasNext()) {
+            files.add(iterator.next().getPath().getName());
```

Comment:
This is discarding most of the file path, so that we're no longer asserting on where each of the files are under the base path.